osgp/platform/osgp-secret-management/src/main/java/org/opensmartgridplatform/secretmanagement/application/endpoints/SoapEndpointDataTypeConverter.java

```diff
@@ -16,100 +16,93 @@
-        TypedSecrets soapTypedSecrets = new TypedSecrets();
+    public TypedSecrets convertToSoapTypedSecrets(final List<TypedSecret> typedSecrets) {
+        final TypedSecrets soapTypedSecrets = new TypedSecrets();
 
-        List<org.opensmartgridplatform.ws.schema.core.secret.management.TypedSecret> soapTypedSecretList =
+        final List<org.opensmartgridplatform.ws.schema.core.secret.management.TypedSecret> soapTypedSecretList =
                 soapTypedSecrets.getTypedSecret();
 
-        for (TypedSecret typedSecret : typedSecrets) {
-            org.opensmartgridplatform.ws.schema.core.secret.management.TypedSecret soapTypedSecret =
-                    encryptAndConvertSoapTypedSecret(
+        for (final TypedSecret typedSecret : typedSecrets) {
+            final org.opensmartgridplatform.ws.schema.core.secret.management.TypedSecret soapTypedSecret = this.encryptAndConvertSoapTypedSecret(
                     typedSecret);
             soapTypedSecretList.add(soapTypedSecret);
         }
 
         return soapTypedSecrets;
     }
 
-    private SecretType convertToSecretType(
-            org.opensmartgridplatform.ws.schema.core.secret.management.SecretType soapSecretType) {
+    public SecretType convertToSecretType(
+            final org.opensmartgridplatform.ws.schema.core.secret.management.SecretType soapSecretType) {
         return SecretType.valueOf(soapSecretType.value());
     }
 
     private org.opensmartgridplatform.ws.schema.core.secret.management.SecretType convertToSoapSecretType(
-            SecretType secretType) {
+            final SecretType secretType) {
         return org.opensmartgridplatform.ws.schema.core.secret.management.SecretType.fromValue(secretType.name());
     }
 
     private org.opensmartgridplatform.ws.schema.core.secret.management.TypedSecret encryptAndConvertSoapTypedSecret(
-            TypedSecret typedSecret) {
-        org.opensmartgridplatform.ws.schema.core.secret.management.TypedSecret soapTypedSecret =
+            final TypedSecret typedSecret) {
+        final org.opensmartgridplatform.ws.schema.core.secret.management.TypedSecret soapTypedSecret =
                 new org.opensmartgridplatform.ws.schema.core.secret.management.TypedSecret();
 
-        String encodedSecret = typedSecret.getSecret();
-        byte[] rawSecret = HexUtils.fromHexString(encodedSecret);
-        Secret secret = new Secret(rawSecret);
-        EncryptedSecret encryptedSecret = encryptionDelegate.encrypt(EncryptionProviderType.RSA, secret, KEY_REFERENCE);
-        soapTypedSecret.setSecret(HexUtils.toHexString(encryptedSecret.getSecret()));
+        //final String encodedSecret = typedSecret.getSecret();
+        final byte[] rsaSecret = typedSecret.getSecret(); //HexUtils.fromHexString(encodedSecret);
+        //final EncryptedSecret encryptedSecret = this.encryptionDelegate.encrypt(EncryptionProviderType.RSA,
+        //        rawSecret, KEY_REFERENCE);
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') This block of commented-out lines of code should be removed. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://ci.opensmartgridplatform.org/sonarqube/coding_rules#rule_key=squid%3ACommentedOutCodeLine)